lib/presentation/home/home_page.dart

```diff
@@ -1,48 +1,53 @@
-import 'package:auto_route/auto_route.dart';
+import 'package:collaction_app/presentation/themes/constants.dart';
```

Comment:
Use relative comments

Couldn't comment on binary, but please remove the `assets/images/ocean.png` and other unused assets, for images, prefer network images so we don't add unnecessary resources.

Reply:
Done.